Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.cpp

```diff
@@ -416,7 +416,8 @@ void CoordinatedGraphicsLayer::setContentsOpaque(bool b)
         m_needsDisplay.completeLayer = true;
         m_needsDisplay.rects.clear();
 
-        addRepaintRect({ { }, m_size });
+        FloatRect layerRect { { }, m_size };
+        addRepaintRect(layerRect);
```

Comment:
Why?

Reply:
Ouch. Looks like the branch lost some context. It was supposed to be:

```cpp
        FloatRect layerRect { { }, m_size };
#if ENABLE(BUFFER_DAMAGE_TRACKING)
        addDamageRegion(layerRect);
#endif
        addRepaintRect(layerRect);
```

Checking where we lost it.

Reply:
It was removed in Adrian's patches - I'll bring it to the very default

Reply:
done